src/styles/variables.less

```diff
@@ -25,12 +40,8 @@
 @color-warning: #feb209;
 @color-danger: #f7403a;
 
-@color-white: #fff;
-
-@color-gray-5: fade(#000, 5%);
-@color-gray-10: fade(#000, 10%);
-@color-gray-25: fade(#000, 25%);
-@color-gray-30: fade(#000, 30%);
+@color-borderColor: @color-gray;
+@color-backgroundColor: @color-ligthGray;
 
 // default
 @color-default-borderColor: #c5c5c5;
```

Comment:
I'm not sure I understand why we couldn't use `@color-mediumGray` as the value of `@color-default-borderColor` since `@color-mediumGray` is `#c5c5c5`.